tests/vmi_multiqueue_test.go

```diff
@@ -59,19 +63,41 @@ var _ = Describe("[sig-compute]MultiQueue", func() {
 		BeforeEach(func() {
 			availableCPUs = libnode.GetHighestCPUNumberAmongNodes(virtClient)
 		})
+
 		It("[test_id:4599]should be able to successfully boot fedora to the login prompt with networking mutiqueues enabled without being blocked by selinux", func() {
-			vmi := tests.NewRandomFedoraVMIWithGuestAgent()
+			const (
```

Comment:
I would create a new test "should be able to run a VM with multiqueue and a mix of virtio and non-virtio interfaces".
Now it is not clear why the second interface is added.

Reply:
Note: no need a mix, an e1000 is enough

The conditions are:
1. Multi queue is enabled
2. There is more than one vCPU
3. There is at least one non virtio interface, (only one interface which is e1000 is good enough)

I just added another interface to an existing test. in order to spin less vms during e2e.

Does it change the picture?
Btw imho testing this corner case is an over kill